tools/cli/adaparse.cpp

```diff
@@ -0,0 +1,299 @@
+#include <cstdlib>
+#include <chrono>
+#include <iostream>
+#include <string_view>
+#include <sys/stat.h>
+#include <fmt/core.h>
+#include <cxxopts.hpp>
+#include <fstream>
+#include <fmt/os.h>
+#include <cerrno>
+
+#include <fmt/os.h>
+#include <typeinfo>
+#include <iostream>
+#include <cxxabi.h>
```

Comment:
Do we need cxxabi.h? It is apparently not a standard header. It breaks the Visual Studio build.

Reply:
Apparently not. I forgot to prune the libraries, and after doing so, all that remains is what is nescessary for it to build (of which cxxabi.h is not one of them).